xml/System.Security/SecureString.xml

```diff
@@ -130,7 +130,7 @@
  Although data stored in a <xref:System.Security.SecureString> instance is more secure than data stored in a <xref:System.String> instance, there are significant limitations on how secure a <xref:System.Security.SecureString> instance is. These include:  
   
  Platform  
- On the Windows operating system, the contents of a <xref:System.Security.SecureString> instance's internal character array is encrypted. However, whether because of missing APIs or key management issues,  encryption is not available on all platforms. Because of this, <xref:System.Security.SecureString> is available on Desktop (Windows only), and not on .NET Core.  
+ On the Windows operating system, the contents of a <xref:System.Security.SecureString> instance's internal character array are encrypted. However, whether because of missing APIs or key management issues,  encryption is not available on all platforms. Because of this limitation, <xref:System.Security.SecureString> may not work correctly on .Net core. 
```

Comment:
> Because of this limitation, <xref:System.Security.SecureString> may not work correctly on .Net core.

That depends on what "correctly" means.  The contents are stored outside of the managed heap in memory marked to not page out (absent hibernation), so it still protects against heap compaction copies.

The contents _are_ only encrypted on Windows (during times that the contents are not being utilized), but the API functions on all platforms.  Note the name of the type is not "EncryptedString".

Reply:
@yevster Can you update this PR with the following text (based on product team feedback):

> Because of this platform dependency, xref:System.Security.SecureString does not encrypt the internal storage on non-Windows platform. Other techniques are used on those platforms to provide additional protection.